File: Jaewon/3) 완주하지 못한 선수/3.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    
    for(int i = 0; i < completion.size(); i++){
        if(participant[i] != completion[i]){
            answer = participant[i];
            return answer;
        }
    }
    answer = participant[participant.size()-1];
    return answer;
}

File: Jaewon/3) 완주하지 못한 선수/3.js
function solution(participant, completion) {
    var answer = '';
    var par = participant.sort();
    var com = completion.sort(); 
    
    for(var i=0;i<par.length;i++){
        if(par[i] !== com[i]){
            answer = par[i];
            return answer;
        }
    }
}